Sources/Models/API/ProductAttributesReadAPI.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ProductAttributesAPIkeysJSON.swift
+//  OpenFoodFacts
+//
+//  Created by Alexander Scott Beaty on 9/29/20.
+//
+
+import Foundation
+
+// MARK: - ProductAttributes keys
+struct ProductAttributesJson {
+    static let ProductAttributesKey = "product_attributes"
+    static let BarcodeKey = "code"
+    static let AttributeGroupsKey = "attribute_groups"
+}
+
+// MARK: - AttributeGroup keys
+struct AttributeGroupJson {
+    static let id = "id"
+    static let NameKey = "name"
+    static let AttributesKey = "attributes"
+}
+
+// MARK: - Attribute type keys
+struct AttributeTypesJson {
```

Comment:
I don't think we should hardcode sections so that we can update them on the fly, or perhaps hardcode some essential ones and have an evolutive fallback for any unsupported section
poke @stephanegigandet

Reply:
What is the hardcoding here? It is very very flexible?

So I do not get your comment.

Reply:
Let's say we want to add a "product recalls" section without updating the app. The design plan behind attributes is that we're able to do that.

Reply:
@aleene: The hardcoding @teolemon is referring to is the `AttributeTypesJson` keys struct. 
I made that in a desperate attempt to make the mapping work, but then I got it to work without those keys. I forgot to remove it in my last commit. 

I don't need them after all so I will remove them. The `attributeGroups` are correctly mapped now without explicitly naming the "Attribute Types," so no need to worry, it is dynamic and the hardcoding is not necessary.

Reply:
OK, I get it. Did not see that struct.